src/datadog/w3c_propagation.cpp

```diff
@@ -190,6 +196,7 @@ void parse_datadog_tracestate(ExtractedData& result, StringView datadog_value) {
     const auto value = range(kv_separator + 1, pair_end);
     if (key == "o") {
       result.origin = std::string{value};
+      std::replace(result.origin->begin(), result.origin->end(), '~', '=');
```

Comment:
You'll appreciate a comment here a few months from now, to remind why this was done.
The corresponding tests don't really point out the what and why of this

Reply:
Will do.